### ISSUE TITLE
BorderColor does not appear to be available for frame. 

### ISSUE THREAD

**Issue by StephanieLakin** [dotnet/docs-maui#1365]
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 04f191eb-6743-c3c6-98d2-de82e088d27f
* Version Independent ID: 04f191eb-6743-c3c6-98d2-de82e088d27f
* Content: [Frame - .NET MAUI](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/frame?view=net-maui-7.0)
* Content Source: [docs/user-interface/controls/frame.md](https://github.com/dotnet/docs-maui/blob/main/docs/user-interface/controls/frame.md)
* Product: **dotnet-mobile**
* Technology: **dotnet-maui**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

**Comment by davidbritch** [comment]
Hi @StephanieLakin 

Is it just intellisense complaining about it, or does it actually produce a build error?

https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.frame.bordercolor?view=net-maui-7.0

**Comment by davidbritch** [comment]
I just tried it - it's definitely present in .NET 7.

**Comment by StephanieLakin** [comment]
Thanks for your quick response, David. It was producing a build error, and I was also missing the TextColor property for a Label. These were not appearing in the dropdown and I was getting an is error stating that the properties were not available for these elements. On building there was then an error for iOS platform version, after resolving that and restarting VS, the properties issue no longer exists, and they now appear in the dropdown. I noticed there were a few extensions waiting to be installed on restart, so maybe something was missing there?  I just got set up with Maui, .net 7 and updated VS 2022 last night and started a Maui tutorial this morning. Thanks again for reaching out.


Get Outlook for Android<https://aka.ms/AAb9ysg>
